### DISCUSSION TITLE
Maintainers and the future of MagmaWM

### DISCUSSION THREAD

**Discussion by electron271** [MagmaWM/MagmaWM#55] [Announcements]
Hi everyone, electron271 (Ave) here, I would like to make a public announcement about the future of MagmaWM. It is quite possible that we may have to put MagmaWM on hold.
My reasoning for this is:
- Our main developer, [HackedOS](https://github.com/HackedOS) has lost their 2fa keys, and has been burnt out
- Our project manager, [Insert5StarName](https://github.com/Insert5StarName) has been busy with irl stuff
- Our tiling developer, [a-weeb-programmer](https://github.com/a-weeb-programmer) has seemingly disappeared, and I have not heard from them in a while
Due to this, I have personally started to lose motivation for MagmaWM. I am planning to start development again (hopefully with the other devs) on June 25th, but I am not sure if that will happen. I am sorry for the inconvenience, and I hope you understand.

If you have any questions, please feel free to ask in the comments or in the Discord.
Thank you for your patience, and have a nice day.

**Comment by HackedOS** [comment]
I got my 2fa back 🤣

**Reply by electron271** [reply]
yipee

**Comment by electron271** [comment]
[]()

**Comment by Phant80m** [comment]
I use arch btw :+1:

**Reply by electron271** [reply]
real

**Comment by Phant80m** [comment]
whats going on now lads?

**Reply by electron271** [reply]
the dream is joever
